modules/phase_field/src/kernels/MatCoupledLaplacian.C

```diff
@@ -0,0 +1,70 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "MatCoupledLaplacian.h"
+
+template <>
+InputParameters
+validParams<MatCoupledLaplacian>()
```

Comment:
This is LaplacianSplit with a predator. MatLaplacianSplit and drive from that class would be my suggestion.